README.md

```diff
@@ -435,6 +436,71 @@ request to `$URL_TO_GET_OIDC_TOKEN`, e.g. `Metadata-Flavor=Google`.
 You can now [use the Auth library](#using-external-identities) to call Google Cloud
 resources from an OIDC provider.
 
+### Accessing resources using X.509 certificate-sourced credentials
+For X.509 certificate-sourced credentials, the authentication library uses an X.509 certificate and private key to prove your application's identity. The certificate has a built-in expiration date and must be renewed to maintain access.
+
+The library constructs a subject token by creating a JSON array containing the base64-encoded leaf certificate, followed by any intermediate certificates from a provided trust chain.
+
+#### Generating Configuration Files for X.509 Federation
```

Comment:
Can this section be renamed something like `Prerequisites: Generating ...` since this seem to me to be moreso describing the commands needed to the credentials?

Using the credentials in the Auth library looks to be in a different section  (#using-external-identities)